output/ISARIC FOLLOWUP/adtl-output_tier1_initial.md

```diff
@@ -0,0 +1,17 @@
+> adtl isaric/parsers/isaric-tier1.toml Tier1FreestandingIni_DATA_2022-06-24_0755.csv
+
+|table          |valid  |total  |percentage_valid|
+|---------------|-------|-------|----------------|
+|subject        |9595   |16046  |59.796834% |
+|visit          |16045  |16046  |99.993768% |
+|observation    |318977 |318977 |100.000000% |
+
+## subject
+
+* 5180: data cannot be validated by any definition
```

Comment:
is this because of required fields missing?

Reply:
Yes - in most cases it's because they don't have an admission date. This is expected as Tier 1 subjects weren't usually admitted to hospital at all (I think that's right @sadiekelly?) I've added it into the optional-fields now, but I think there's a bug in adtl where it's not updating the "anyOf" requirements in the schema so it still gives a low pass rate.

Reply:
This is corrected in the updated version of adtl to a 96% pass rate